src/containerapp/azext_containerapp/_clients.py

```diff
@@ -791,21 +791,6 @@ def check_name_availability(cls, cmd, resource_group_name, name, name_availabili
         r = send_raw_request(cmd.cli_ctx, "POST", request_url, body=json.dumps(name_availability_request))
         return r.json()
 
-    @classmethod
-    def get_auth_token(cls, cmd, resource_group_name, name):
```

Comment:
why delete it?

Reply:
Migrate to SDK, use client.get_auth_token.